fast_barcode_scanner/lib/src/camera_controller.dart

```diff
@@ -207,7 +207,6 @@ class _CameraController implements CameraController {
     } catch (error) {
       state._error = error;
       events.value = ScannerEvent.error;
```

Comment:
Could we replace these removed `rethrow` with debugLog to catch error case later on

Reply:
state._error and events.value = ScannerEvent.error will cover the error case in UI: 
https://github.com/gojitsucom/fast_barcode_scanner/blob/159c0309e0a270d36bad02d0833c6a78f56073da/fast_barcode_scanner/lib/src/barcode_camera.dart#L121-L125
We should add the log in the onError callback